### PR TITLE
Replace raw SSH commands with `gds govuk connect` equivalents

### PR DESCRIPTION
**Pull request by issyl0**
- Further to #2462.

https://trello.com/c/ZBGHVlTr/132-deprecate-and-delete-govukcli

### CODE REVIEW

File: source/manual/alerts/onsite-backups.html.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ It's likely that this is failing because one of the backups it's trying to colle
 To rerun an individual backup:
 
 ```
-ssh backup-1.management.production
+gds govuk connect ssh -e production backup
```
[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/alerts/onsite-backups.html.md:20]: Ah, good point. But I'm wary of using actual hostnames in AWS in docs, 'cuz they change so regularly. Maybe we could say `<the hostname from the alert>`?

File: source/manual/howto-upload-an-asset-to-asset-manager.html.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ scp my_file.jpg backend-1.backend.production:/tmp/
 Then SSH to the same machine and run the upload command:
 
 ```
-ssh backend-1.backend.production
+gds govuk connect ssh -e production backend
```
[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/howto-upload-an-asset-to-asset-manager.html.md:25]: You'd better get on and make SCP work then. 😉 :trollface:

File: source/manual/sidekiq.html.md [hunk 3]
```diff
@@ -29,31 +29,14 @@ that can display the current state of a [Sidekiq] installation. We have
...
-```
-
-Or on AWS:
-
-```bash
-$ ssh $(ssh integration "govuk_node_list --single-node -c backend").integration -CNL 9000:127.0.0.1:3211
-```
-
-or using `govuk-connect`:
+[Sidekiq] queues. To access the Sidekiq monitoring UI for Production AWS,
```
[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/sidekiq.html.md:32]: Done!

File: source/manual/alerts/onsite-backups.html.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (benthorner) [reply source/manual/alerts/onsite-backups.html.md]: Perhaps something that resembles the hostname? `ip-1-2-3-4.blah.blah`? (So it's clear we're talking about the full, long thing.)

[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/alerts/onsite-backups.html.md]: Does https://github.com/alphagov/govuk-developer-docs/pull/2470/commits/7efa4bddee97c698149bde02ac577768832283da help?

File: source/manual/howto-upload-an-asset-to-asset-manager.html.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/howto-upload-an-asset-to-asset-manager.html.md:25]: I will back out this commit.

[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/howto-upload-an-asset-to-asset-manager.html.md]: Done.

File: source/manual/alerts/onsite-backups.html.md [hunk 4]
```diff
@@ -17,7 +17,7 @@ It's likely that this is failing because one of the backups it's trying to colle
 To rerun an individual backup:
 
 ```
-ssh backup-1.management.production
+gds govuk connect ssh -e production <backup-hostname-from-alert>
```
Comment (issyl0) [inline source/manual/alerts/onsite-backups.html.md:20]:
```suggestion
gds govuk connect ssh -e production <backup-hostname-from-alert> (for example, ip-1-2-3-4.eu-west-1.compute.internal)
```

File: source/manual/alerts/onsite-backups.html.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (issyl0) [reply source/manual/alerts/onsite-backups.html.md]: How is https://github.com/alphagov/govuk-developer-docs/pull/2470/files#r409629931?

**Pull request merged**